analysis/analyse-cpu.js

```diff
@@ -2,6 +2,7 @@
 
 const summary = require('summary')
 const tf = require('@tensorflow/tfjs-core')
+require('@tensorflow/tfjs-backend-cpu')
```

Comment:
this is not needed

Reply:
Without this the tests were failing with the error: `No backend found in registry`. More details in https://github.com/tensorflow/tfjs/issues/3352. Is there is a better way how to resolve this error?

Reply:
Sorry i misread, that's ok